src/commands/pullzone.ts

```diff
@@ -70,8 +72,26 @@ async function purgeCache(this: Config, opts: OptionsPurgeCache) {
           const { id, status } = res.value as PullZonePurgeCacheResponse
           return { id, status }
         })
-        .map(({ id, status }) => ({ idZone: id, purged: `${status === 204 ? 'Ok' : 'Error'} (${status})` }))
+        .reduce<Record<string, unknown>[]>((lines, { id, status }) => {
+          isOk = isOk && status === 204
```

Comment:
Honestly, instead of using `isOk` to check if an error occurred, I would use `failedZoneIds.length`

Reply:
Fair.. was in a rush